keps/sig-network/3070-reserved-service-ip-range/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: Reserve Service IP Range For Dynamic and Static IP Allocation
+kep-number: 3070
+authors:
+  - "@aojea"
+owning-sig: sig-network
+participating-sigs:
+status: provisional
```

Comment:
I am haappy to merge this as provisional, but it feels just about ready for implementable, modulo PRR.

Reply:
yep